src/pages/LandingPage.tsx

```diff
@@ -1,15 +1,12 @@
+import StandardLayout from '@/templates/StandardLayout';
 import { LandingGrid } from '@/organisms/LandingGrid';
-import { SplashVideo, SplashVideoContainer } from '@/organisms/SplashVideo'
-import { Footer } from '@/organisms/Footer';
+import { SplashVideo } from '@/organisms/SplashVideo'
 
 export default function LandingPage() {
   return (
-    <>
-      <SplashVideoContainer>
-        <SplashVideo />
-      </SplashVideoContainer>
+    <StandardLayout>
+      <SplashVideo />
       <LandingGrid />
-      <Footer />
```

Comment:
Same argument for Header component goes for Footer. I think I meant to put this one a level higher (but didn't because at the time I was solely focusing on LandingPage and I don't think the router was set up yet) but because it's shared across all pages it's not necessarily considered as part of StandardLayout/LayoutContent